hack/verify-crd-gen.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
nit: `#!/usr/bin/env bash` is more portable.

Reply:
Updated the path as suggested with the latest push.